etc/scripts/json2csv.py

```diff
@@ -246,7 +244,7 @@ def cli(json_input, csv_output):
 
     JSON_INPUT is either a ScanCode json format scan or the data.json file from a ScanCode html-app format scan.
 
-    Paths will be prefixed with '/code/' to provide a common base directory for scanned resources.
+    #Paths will be prefixed with '/code/' to provide a common base directory for scanned resources.
```

Comment:
@JonoYang so if there is a directory named home then do nothing to path or else prepend it by /code.
If that you mean in issue 524 then I have done it